### DISCUSSION TITLE
Why I recieved this output:

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#760] [Q&A] [answered]
CODE:

```

import pandas as pd 
"""
Routes and views for the flask application.
"""

from datetime import datetime
from xmlrpc.client import DateTime
from flask import render_template
from gimlaot import app

@app.route('/home')
def home():
    """Renders the home page."""
    return render_template(
        'index.html',
        title='Home Page',
        year=datetime.now().year,
    )
@app.route('/')
def Index():
    import falconpy
    H=falconpy.Hosts(client_id="__",client_secret="___")
#    t=H.QueryDevicesByFilter(filter='*',limit=555)
    n=datetime.now()
       
    detect=falconpy.Detects(client_id="__",client_secret="___")
    df=pd.DataFrame(data=(detect.get_detect_summaries(ids=detect.query_detects(sort="last_behavior|asc",filter=f"first_behavior:>='{datetime.fromtimestamp(n.timestamp()-30*24*86400).isoformat()}'",limit=333)["body"]["resources"])))
    assd=[detect.get_detect_summaries(ids=detect.query_detects(sort="last_behavior|asc",filter=f"first_behavior:>='{datetime.fromtimestamp(n.timestamp()-30*24*86400).isoformat()}'",limit=333)["body"]["resources"][0])]
    tdc=pd.DataFrame(detect.get_detect_summaries(ids=detect.query_detects(sort="last_behavior|asc",filter=f"first_behavior:>='{datetime.fromtimestamp(n.timestamp()-30*24*86400).isoformat()}'",limit=333)["body"]["resources"][0]))
    for s in range(len(detect.get_detect_summaries(ids=detect.query_detects(sort="last_behavior|asc",filter=f"first_behavior:>='{datetime.fromtimestamp(n.timestamp()-30*24*86400).isoformat()}'",limit=333)["body"]["resources"]))):
        assd.append(detect.get_detect_summaries(ids=detect.query_detects(sort="last_behavior|asc",filter=f"first_behavior:>='{datetime.fromtimestamp(n.timestamp()-30*24*86400).isoformat()}'",limit=333)["body"]["resources"][s+1]))
    j=""
    c=0
    a={}
    for i in assd:
        for j in i["body"]["resources"]:
            c=c+1
            
            a[str(c)]=j["device"]["hostname"]


    
    return  pd.DataFrame(data=a).to_html()
 
@app.route('/contact')
def contact():
    """Renders the contact page."""
    return render_template(
        'contact.html',
        title='Contact',
        year=datetime.now().year,
        message='Your contact page.'
    )

@app.route('/about')
def about():
    """Renders the about page."""
    return render_template(
        'about.html',
        title='About',
        year=datetime.now().year,
        message='Your application description page.'
    )

```

**Comment by jshcodes** [comment, marked as answer]
This is pretty far out of scope, but I'll give it a shot. Reviewing the error message in your IDE it appears you are passing scalars to the Dataframe.  Since there is no index, you're given an error.  Googling about, several folks who hit this error pass the keyword `index=[0]` and provide a dummy index.  Depending on your goals, this may or may not be the correct solution.

**Comment by NSH531** [comment]
Someone here is understanding at pandas?

בתאריך יום חמישי, 25 באוגוסט 2022, מאת Joshua Hiller <
***@***.***>:

> This is pretty far out of scope, but reviewing the error message in your
> IDE it appears you are passing scalars to the Dataframe. Since there is no
> index, you're given an error. Googling about, several folks who hit this
> error pass the keyword index=[0] and provide a dummy index. Depending on
> your goals, this may not be the correct solution.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CrowdStrike/falconpy/discussions/760#discussioncomment-3475300>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABNBJWOHTGWD54YFK4D4MQTV257SJANCNFSM57SNFC5Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
<https://netanel.ml>

**Reply by jshcodes** [reply]
Hi @NSH531 -

If passing an index doesn't help, you can probably get more assistance from the pandas issue board located here: https://github.com/pandas-dev/pandas/issues

**Reply by NSH531** [reply]
I sent issue :
https://github.com/pandas-dev/pandas/issues/48253

**Reply by NSH531** [reply]
AN ANOTHER ISSUE 
 https://github.com/pandas-dev/pandas/issues/48256



TNX
NATE

**Comment by NSH531** [comment]
Someone is understad on flask?


-- 
<https://netanel.ml>

**Reply by jshcodes** [reply]
Hi @NSH531 -

Please keep questions posted to this discussion board focused on topics related to FalconPy or the CrowdStrike APIs.  This will help make sure this content stays relevant to the community at large. :smile:

[Discussion board usage](https://github.com/CrowdStrike/falconpy/discussions/638)

**Comment by NSH531** [comment]
I reshared the issues that relevant for another python libraries that I use
them.:
Pandas
Flask

Therefore,  I censored apikey and secret

Tnx,
Nate

בתאריך יום ו׳, 26 באוג׳ 2022, 15:52, מאת Joshua Hiller ‏<
***@***.***>:
